project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my_groups.inc

```diff
@@ -61,6 +61,7 @@ function c4m_features_homepage_my_groups_content_type_render($subtype, $conf, $a
   $variables = array(
     'groups' => $rendered_groups,
     'link' => url('groups', $link_options),
+    'num_groups' => $count,
```

Comment:
Check this one:
https://github.com/capacity4dev/capacity4more/pull/1501/files#diff-e90e2a247400ce9fd778edea9df3d00eR59